openroad.bzl

```diff
@@ -891,7 +874,7 @@ def build_openroad(
                 write_stage_config(
                     name = target_name + "_gui_" + stage + "_config",
                     stage = stage,
-                    srcs = stage_cfg_srcs,
+                    srcs = stage_cfg_sources["synth"] + stage_cfg_sources["synth_sdc"],
```

Comment:
`stage_args["synth_sdc"]` has a `$(location <label>)` expansion of `stage_cfg_sources["synth_sdc"]`. `lefs_args` refers to `ADDITIONAL_LEFS`, but as this is not `$(location <label>)` expanded, we do not need to encode the dependency. Should these not be treated the same way?

Reply:
Yes, thanks. However it is unclear to me why we should use `$(location <label>)` for some paths, and `$(BINDIR)` for others?

Reply:
I don't think there is a strict rule, `$(BINDIR)` gives more freedom and `$(location <label>)` should be more robust (especially for targets from external repositories). In case on `ADDITIONAL_LES/LIBS` we can use both, I believe. You can change it to `location`, it will probably be cleaner and more consistent.